pkg/pquerier/collector.go

```diff
@@ -204,3 +215,57 @@ func downsampleRawData(ctx *selectQueryContext, res *qryResults,
 
 	return lastT, lastV, nil
 }
+
+func aggregateClientAggregatesCrossSeries(ctx *selectQueryContext, res *qryResults, previousPartitionLastTime int64, previousPartitionLastValue float64) (int64, float64, error) {
+	ctx.logger.Debug("using Client Aggregates Collector for metric %v", res.name)
+	it := newRawChunkIterator(res, nil).(*rawChunkIterator)
+
+	var lastT int64
+	var lastV float64
+
+	var previousPartitionEndBucket int
+	if previousPartitionLastTime != 0 {
+		previousPartitionEndBucket = int((previousPartitionLastTime-ctx.queryParams.From)/ctx.queryParams.Step) + 1
+	}
+	maxBucketForPartition := int((res.query.partition.GetEndTime() - ctx.queryParams.From) / ctx.queryParams.Step)
+	if maxBucketForPartition > ctx.getResultBucketsSize() {
+		maxBucketForPartition = ctx.getResultBucketsSize()
+	}
+
+	for currBucket := previousPartitionEndBucket; currBucket < maxBucketForPartition; currBucket++ {
+		currBucketTime := int64(currBucket)*ctx.queryParams.Step + ctx.queryParams.From
+
+		if it.Seek(currBucketTime) {
+			t, v := it.At()
+			if t == currBucketTime {
+				for _, col := range res.frame.columns {
+					if col.GetColumnSpec().metric == res.name {
+						col.SetDataAt(currBucket, v)
```

Comment:
If we're going to ignore errors, we should use `_ = ` notation to explicitly do so (I'm surprised the linter is ok with this - I think that's not the case in Nuclio).